app/Models/CollectionType.php

```diff
@@ -26,8 +26,8 @@ public static function arrayForSelect()
     return self::pluck('name','id')->toArray();
   }
 
-  public static function formattedName($collectionType)
+  public function formattedName()
   {
-    return $collectionType === null ? '--' : $collectionType->name;
+    return $this === null ? '--' : $this->name;
```

Comment:
Sorry, I was wrong. I guess this won't work since collectiontypes are not always present.

Reply:
Haha! I knew there must have been a reason i did it that way...I totally forgot about that. I could still make it a class method I think

Reply:
Ha, probably not worth it. Or you could change it to a view helper, maybe.

Reply:
Oh, that works!